src/components/File.jsx

```diff
@@ -69,7 +67,7 @@ class File extends Component {
       this.props.onOpen(id)
     } else {
       // TODO Handle files opening throught the app instead of doing it throught the browser
-      window.open(`${STACK_URL}/files/download/${id}`, '_blank')
+      window.open(`${cozy._url}/files/download/${id}`, '_blank')
```

Comment:
Yes we are!

And in the *mobile* app, we store that url in the global state, because we need it to init the `cozy-client-js` instance.